graphql_api/types/repository/repository.py

```diff
@@ -283,6 +283,15 @@ def resolve_flags_measurements_backfilled(repository: Repository, info) -> bool:
     return dataset.is_backfilled()
 
 
+@repository_bindable.field("isATSConfigured")
+def resolve_is_ats_configured(repository: Repository, info) -> bool:
+    if not repository.yaml or "flag_management" not in repository.yaml:
```

Comment:
Also, have you double checked if you want the repository.yaml or some other yaml value? Check this fn out, `UserYaml.get_final_yaml(...)`, the yaml can be from a commit, repo and/or org, so I'd double check on your end

Reply:
It looks like it is on the [repository](https://github.com/codecov/codecov-api/commit/6ad32c93f5a2afcca6221c98a3acbf31b92dd7b2#diff-3e878d9bdc47f29a0ab909a7db939acf08d2d2f1aaba81e6f897b32361fa40db). Thanks for the caution here.